contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -817,7 +827,7 @@ static VALUE rb_trilogy_query(VALUE self, VALUE query)
 {
     struct trilogy_ctx *ctx = get_open_ctx(self);
 
-    StringValue(query);
```

Comment:
Can we keep this StringValue check? I'm not convinced that `rb_str_export_to_enc` does the check for a T_STRING that we need, (I think that check needs to happen before calling `rb_str_export_to_enc` based on `ExportStringValue` as an example)

Reply:
Absolutely, good call!